main/views.py

```diff
@@ -183,7 +339,10 @@ def dispatch(self, request, *args, **kwargs):
         if not request.user.is_authenticated:
             return self.handle_no_permission()
         # if user already has an active membership then return to profile page
-        if request.user.supporter_status() == 'Pending' or request.user.supporter_status() == 'Approved':
+        if (request.user.supporter_status() == 'Pending'
+            or request.user.supporter_status() == 'Approved'
+            or request.user.supporter_status() == 'Emailed'):
```

Comment:
The indentation here is causing a CI Lint error:
`./main/views.py:344:13: E129 visually indented line with same indent as next logical line`

Presumably it should be on the same line or indented in line with the if?